indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -830,3 +830,44 @@ dd > .i-has-action {
         padding: 0.25em 0;
     }
 }
+
+
+.color-palette {
+    background: #FFF;
+    background-clip: padding-box;
+    border: thin solid rgba(0, 0, 0, .15);
+    border-radius: 4px;
+    box-shadow: 0 6px 12px rgba(0, 0, 0, .175);
+    display: none;
+    margin-top: 0.5em;
+    padding: 3px;
+    position: absolute;
+    z-index: 200;
+
+    table {
+        border-spacing: 8px;
+    }
+
+    &:before {
+        transform: rotate(45deg);
+        content: '';
+        top: -0.2rem;
+        position: absolute;
+        width: .6em;
+        height: .6em;
+        box-shadow: -1px -1px 0 1px rgba(34, 36, 38, .15);
```

Comment:
Another arbitrary color here. Doesn't any of the colors in `_palette.scss` fit here? Have you tried with `lighten(..., $color-variation)` and `darken(..., $color-variation)`?